File: sdkwork-backend-sdk-typescript/src/types/plus-file-content-vo.ts
import type { FileContentObject } from './file-content-object';

/** æä»¶åå®¹VOï¼ä¸PlusFileä¸å¯¹ä¸å¯¹åºï¼å­å¨æä»¶çææ¬åå®¹åç¼ç ä¿¡æ¯ï¼ */
export interface PlusFileContentVO {
  /** åå»ºæ¶é´ï¼å®ä½é¦æ¬¡æä¹åæ¶è®¾ç½® */
  createdAt?: string;
  /** æåæ´æ°æ¶é´ï¼å®ä½ä¿®æ¹æ¶æ´æ° */
  updatedAt?: string;
  /** ä¸»é®IDï¼ç±æ°æ®åºèªå¨çæ */
  id?: number;
  /** éç¨å¯ä¸æ è¯ç¬¦UUID */
  uuid?: string;
  /** å³èæä»¶ID(å¤é®å³èplus_file.id) */
  fileId?: number;
  /** å³èæä»¶UUID(å¤é®å³èplus_file.uuid) */
  fileUuid?: string;
  /** æç¤ºè¯ */
  prompt?: string;
  /** æä»¶ç¼ç æ ¼å¼(å¦UTF-8ãGBKãISO-8859-1ç­) */
  encoding?: string;
  /** æä»¶ææ¬åå®¹ */
  content?: FileContentObject;
}

File: sdkwork-backend-sdk-java/src/main/java/com/sdkwork/backend/model/PlusFileContentVO.java
package com.sdkwork.backend.model;

public class PlusFileContentVO {
    private String createdAt;
    private String updatedAt;
    private Integer id;
    private String uuid;
    private Integer fileId;
    private String fileUuid;
    private String prompt;
    private String encoding;
    private FileContentObject content;

    public String getCreatedAt() {
        return this.createdAt;
    }
    
    public void setCreatedAt(String createdAt) {
        this.createdAt = createdAt;
    }

    public String getUpdatedAt() {
        return this.updatedAt;
    }
    
    public void setUpdatedAt(String updatedAt) {
        this.updatedAt = updatedAt;
    }

    public Integer getId() {
        return this.id;
    }
    
    public void setId(Integer id) {
        this.id = id;
    }

    public String getUuid() {
        return this.uuid;
    }
    
    public void setUuid(String uuid) {
        this.uuid = uuid;
    }

    public Integer getFileId() {
        return this.fileId;
    }
    
    public void setFileId(Integer fileId) {
        this.fileId = fileId;
    }

    public String getFileUuid() {
        return this.fileUuid;
    }
    
    public void setFileUuid(String fileUuid) {
        this.fileUuid = fileUuid;
    }

    public String getPrompt() {
        return this.prompt;
    }
    
    public void setPrompt(String prompt) {
        this.prompt = prompt;
    }

    public String getEncoding() {
        return this.encoding;
    }
    
    public void setEncoding(String encoding) {
        this.encoding = encoding;
    }

    public FileContentObject getContent() {
        return this.content;
    }
    
    public void setContent(FileContentObject content) {
        this.content = content;
    }
}
